src/main/scala-2.11/markets/clearing/engines/CDAMatchingEngine.scala

```diff
@@ -15,24 +15,208 @@ limitations under the License.
 */
 package markets.clearing.engines
 
-import java.util.UUID
-
-import markets.clearing.strategies.BestLimitPriceFormationStrategy
-import markets.orders.orderings.PriceOrdering
+import markets.clearing.engines.matches.{Match, PartialMatch, TotalMatch}
+import markets.orders.limit.{LimitBidOrder, LimitAskOrder, LimitOrderLike}
+import markets.orders.market.{MarketOrderLike, MarketAskOrder, MarketBidOrder}
 import markets.orders.{AskOrder, BidOrder, Order}
+import markets.orders.orderings.PriceOrdering
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1519375750/issues/source?bid=2921361&fileBranchId=3008855#l22)